static/app/components/assistant/getGuidesContent.tsx

```diff
@@ -348,6 +348,26 @@ export default function getGuidesContent(orgSlug: string | null): GuidesContent
         },
       ],
     },
+    {
+      guide: 'release_stages',
+      requiredTargets: ['release_stages'],
```

Comment:
Should we add some sort of `dateThreshold: new Date(2021, 6, 1)` there so that we don't pollute the initial onboarding experience?

Reply:
I agree, we should probably add the same date threshold as semver.